code/API_definitions/Application_Endpoint_Discovery_API.yaml

```diff
@@ -0,0 +1,596 @@
+---
+openapi: 3.0.3
+info:
+  title: Application Endpoint Discovery API
+  version: 0.1.0-wip
+  description: |
+    # Connect to the Nearest Application Endpoint
+    ---
+    # Overview
+    The Application Discovery API extends beyond the capabilities of the
+    Simple Edge Discovery API by not only locating the nearest Edge Cloud Zone
+    but also directly linking to the application endpoints within those zones.
```

Comment:
I think we should replace the zones with Edge Cloud Zones as it is the formal definition we have.

Reply:
Agree